frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx

```diff
@@ -22,9 +23,7 @@ type MinimalDeploymentAlertProps = {
   isInternalMode?: boolean;
 };
 
-export const MinimalDeploymentAlert: React.FC<MinimalDeploymentAlertProps> = ({
-  isInternalMode,
-}) => (
+export const MinimalDeploymentAlert: React.FC<MinimalDeploymentAlertProps> = () => (
```

Comment:
```suggestion
export const MinimalDeploymentAlert: React.FC = () => (
```
Lets remove the type declaration as well.

Reply:
ack. Missed it